src/tests/packages/utils.unmocked.test.ts

```diff
@@ -0,0 +1,63 @@
+import { decodePngMetadata, decodeSvgMetadata } from "../../data/image";
+import { ImportedDataState } from "../../data/types";
+import * as utils from "../../packages/utils";
+import { API } from "../helpers/api";
+
+describe("embedding scene data", () => {
+  describe("exportToSvg", () => {
```

Comment:
Do we need a separate describe block for this ? Instead the it block could be renamed to `shouldn't modify the elements when exporting to svg`

Reply:
doesn't matter